presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/EnvironmentUp.java

```diff
@@ -45,13 +46,27 @@
     @Inject
     public EnvironmentOptions environmentOptions = new EnvironmentOptions();
 
+    @Inject
+    public EnvUpOptions envUpOptions = new EnvUpOptions();
+
     private Module additionalEnvironments;
 
     public EnvironmentUp(Extensions extensions)
     {
         this.additionalEnvironments = requireNonNull(extensions, "extensions is null").getAdditionalEnvironments();
     }
 
+    public static class EnvUpOptions
```

Comment:
This should actually be `EnvironmentUpOptions` (i will fix https://github.com/prestosql/presto/pull/2747 too)

i am adding a fixup with this